SnowSQL/public/Invoke-SnowSql.ps1

```diff
@@ -44,6 +44,12 @@ function Invoke-SnowSql
         [string[]]
         $Query = '!help',
 
+        # Timeout query
+        [Parameter(ParameterSetName = 'QueryCred')]
+        [Parameter(ParameterSetName = 'QueryConnection')]
+        [int]
+        $timeout = 10,
```

Comment:
Should this have a default? if so, is this too short for the default? People may have queries that run longer than 10 seconds already.

This should be capitalized to match the other parameters.

Reply:
Well seen for the capital letters!
This is the connection attempt time, not the request time (as I understand it). This avoids waiting 120sec when you have an identifier or endpoint error.